data/packrat/artifacts.json

```diff
@@ -0,0 +1,2052 @@
+            ]
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "application": "ICQ",
+    "category": "chats",
+    "file_artifact": [
+      {
+        "filetypes": "logins",
+        "path": "AppData",
+        "dir": "ICQ",
+        "artifact": "Owner.mdb",
+        "description": "ICQ's saved Username & Passwords",
+        "credential_type": "text",
+        "regex_search": [
+          {
+            "extraction_description": "Searches for credentials (USERNAMES/PASSWORDS)",
+            "extraction_type": "credentials",
+            "regex": [
+              "(?i-mx:password.*)",
+              "(?i-mx:username.*)"
+            ]
+          },
+          {
+            "extraction_description": "searches for Email TO/FROM address",
+            "extraction_type": "Email addresses",
+            "regex": [
+              "(?i-mx:to:.*)",
+              "(?i-mx:from:.*)"
+            ]
+          }
+        ]
+      },
+      {
+        "filetypes": "chat_logs",
+        "path": "AppData",
+        "dir": "ICQ",
+        "artifact": "Messages.mdb",
+        "description": "ICQ's chat logs",
+        "credential_type": "text",
+        "regex_search": [
+          {
+            "extraction_description": "Searches for credentials (USERNAMES/PASSWORDS)",
+            "extraction_type": "credentials",
+            "regex": [
+              "(?i-mx:password.*)",
+              "(?i-mx:username.*)"
+            ]
+          },
+          {
+            "extraction_description": "searches for Email TO/FROM address",
+            "extraction_type": "Email addresses",
+            "regex": [
+              "(?i-mx:to:.*)",
+              "(?i-mx:from:.*)"
+            ]
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "application": "miranda",
+    "category": "chats",
+    "file_artifact": [
+      {
+        "filetypes": "logins",
+        "path": "AppData",
+        "dir": "Miranda",
+        "artifact": "Home.dat",
+        "description": "Miranda's multi saved chat protocol Username, (coded Passwords)",
+        "credential_type": "text",
+        "regex_search": [
+          {
+            "extraction_description": "Searches for credentials (USERNAMES/PASSWORDS)",
+            "extraction_type": "credentials",
+            "regex": [
+              "(?i-mx:password.*)",
+              "(?i-mx:username.*)"
+            ]
+          },
+          {
+            "extraction_description": "searches for Email TO/FROM address",
+            "extraction_type": "Email addresses",
+            "regex": [
+              "(?i-mx:to:.*)",
+              "(?i-mx:from:.*)"
+            ]
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "application": "nimbuzz",
+    "category": "chats",
+    "file_artifact": [
+      {
+        "filetypes": "logins",
+        "path": "AppData",
+        "dir": "nimbuzz",
+        "artifact": "nimbuzz.log",
+        "description": "Username&Password - user phone number",

```

Comment:
and password? looks like it only looks for user (although it could be a combined string or something similar)

Reply:
https://github.com/rapid7/metasploit-framework/blob/76954957c740525cff2db5a60bcf936b4ee06c42/modules/post/windows/gather/credentials/nimbuzz.rb

Looks through the registry instead of the logs.   Not sure if this is looking for someone chatting a username/pw combo like in other modules, or the user's login data